.github/workflows/createTagFromDispatch.yml

```diff
@@ -0,0 +1,20 @@
+name: Create tag (notified from dispatch event)
```

Comment:
should the filename be 
create-tag-from-dispatch.yml
to be consistent with the other workflow file naming convention here ?
I see the convention is different on Radium-Release but the same on Radium-Apps ... it's not mandatory, but cleaner to have consistency.

Reply:
Hum right, I've followed the naming convention of Radium-Releases, but true, it's better to stick to Radium-Engine convention.

Reply:
Done.